Dialog/TilesetEditDialog.cpp

```diff
@@ -581,6 +591,28 @@ void TilesetEditDialog::OverwriteATile8x8InTile8x8MapAndUpdateTile16Map(int posI
     }
 }
 
+/// <summary>
+/// Update Info Textbox
+/// </summary>
+void TilesetEditDialog::UpdateInfoTextBox()
+{
+    // clean up textbox
+    ui->textEdit_Infos->setText("");
+
+    // Generate new Infos
+    QString output;
+    output += tr("Unused Palette(s): (don't forget to clean up those trash Tile16 manually)\n");
```

Comment:
trash -> unused

Reply:
“Unused Palette(s): (don't forget to clean up those unused Tile16 to free up the room for new palettes.”